packages/realtime_client/lib/src/realtime_channel.dart

```diff
@@ -410,6 +410,17 @@ class RealtimeChannel {
     );
   }
 
+  /// Sets up a listener for realtime system events for debugging purposes.
+  RealtimeChannel onSystemEvents(
+    void Function(Map<String, dynamic> payload) callback,
+  ) {
+    return onEvents(
+      'system',
+      ChannelFilter(),
+      (payload, [ref]) => callback(Map<String, dynamic>.from(payload)),
```

Comment:
Are you sure it's always a Map? The js library says it's `any` https://github.com/supabase/realtime-js/pull/422/files#diff-9cd593cbb8706c342a9ff3945900565d4e5e712c5a4cd14c0e12c0326bee43f9R419

Reply:
Oops, good call.